entries/c/coindcx.com.json

```
@@ -0,0 +1,15 @@
+{
+    "CoinDCX": {
+      "domain": "coindcx.com",
+      "tfa": [
+        "sms",
+        "email",
+        "totp"
+      ],
+      "documentation": "https://support.coindcx.com/hc/en-gb/articles/360003181855-How-to-secure-your-account-with-Two-factor-Authentication-2FA-",
+      "keywords": [
+        "cryptocurrencies"
+      ]
```

Looks like the products are mainly targeted to India
```suggestion
      ],
      "regions": [
        "in"
      ]
```